### DISCUSSION TITLE
Help finding wired network card (noob?)

### DISCUSSION THREAD

**Discussion by deeckste** [Security-Onion-Solutions/securityonion#5728] [Unsupported Versions] [answered]
Hello,

I am a noob to the SecurityOnion and trying to get it setup.  I am struggling to find a wired network card that supports promiscuous mode.  I noticed in the documentation that Intel works well, but I cant figure out which Intel card will work.  Looking for something that supports gigabit.  Many thanks in advance for any guidance.

**Comment by FritzLott** [comment, marked as answer]
I am using Security Onion to monitor my home network.  I bought a [Dell SFF](https://www.microcenter.com/product/628481/dell-optiplex-7020-sff-desktop-computer-(refurbished).  I bought an [Intel NIC](https://www.microcenter.com/product/360498/intel-expi9301ct-gigabit-pcie-x1-adapter).  The installation from the ISO worked easily and immediately found the added Intel NIC.  The CPU is running at about 30%, so I am hopeful that the Dell will be adequate for my home network after I install the remaining packages.

My understanding is that all network cards can run in promiscuous mode.  The difficulty is finding a card that works with the ISO installation.

**Reply by deeckste** [reply]
Oh interesting, I did know all network cards can support promiscuous mode.  Thanks for sharing the NIC details, that helps.  Many thanks!
